serverless/admin_guide/upgrading-serverless.adoc

```diff
@@ -10,7 +10,7 @@ If you have installed a previous version of {ServerlessProductName}, follow the
 
 [IMPORTANT]
 ====
-Before upgrading to the latest Serverless release, you must remove the community Knative Eventing Operator if you have previously installed it. Having the Knative Eventing Operator installed will prevent you from being able to install the latest version of Knative Eventing.
+Before upgrading to the latest Serverless release, you must remove the community Knative Eventing Operator if you have previously installed it. Having the Knative Eventing Operator installed will prevent you from being able to install the latest version of Knative Eventing using the {ServerlessOperatorName}.
```

Comment:
If you made any change in the previous use, then friendly reminder for here too.